CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 - Enforce rate limits for monitor check-ins. ([#2065](https://github.com/getsentry/relay/pull/2065))
 - Allow rate limits greater than `u32::MAX`. ([#2079](https://github.com/getsentry/relay/pull/2079))
 - Do not drop envelope when client closes connection. ([#2089](https://github.com/getsentry/relay/pull/2089))
+- Sample only transaction events instead of sampling both transactions and errors. ([#2130](https://github.com/getsentry/relay/pull/2130))
```

Comment:
Please, move this to `Unreleased` section.

Reply:
Good catch! Thanks